jdbc-hikari/src/main/java/io/micronaut/configuration/jdbc/hikari/package-info.java

```diff
@@ -18,8 +18,11 @@
  */
 @Configuration
 @Requires(classes = HikariDataSource.class)
```

Comment:
Maybe add requires to BasicJdbcConfiguration and that should be enough for all implementations

Reply:
I'll try 🤔

Reply:
@dstepanov No, the `@Requires` doesn't seem to carry through...  In my tests with Hikari, we still end up with a `DatasourceConfiguration` and a `HikariUrlDataSource` bean